src/main/java/org/team1540/elmo/ejectors/EjectLower.java

```diff
@@ -0,0 +1,15 @@
+package org.team1540.elmo.ejectors;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+
+public class EjectLower extends ParallelCommandGroup {
```

Comment:
When would you want to only eject the lower two and not the upper one?

Reply:
maybe in auto? idk